perf-regression/Makefile.subdir

```diff
@@ -1,5 +1,12 @@
 if HAVE_MARGO
-bin_PROGRAMS += perf-regression/margo-p2p-latency perf-regression/margo-p2p-bw perf-regression/margo-p2p-vector
+bin_PROGRAMS += perf-regression/margo-p2p-latency perf-regression/margo-p2p-bw perf-regression/margo-p2p-vector perf-regression/gpu-margo-p2p-bw
+
+#Remove -pthread doesn't work for nvcc
+CUDA_CFLAGS := $(subst -pthread,,$(CFLAGS))
+
+perf-regression/gpu-margo-p2p-bw: perf-regression/gpu-margo-p2p-bw.cu
+	nvcc -I. $(CUDA_CFLAGS) $(LIBS) -I/lus/theta-fs0/software/thetagpu/openmpi-4.0.5/include -L/lus/theta-fs0/software/thetagpu/openmpi-4.0.5/lib -lmpi $< -o $@
```

Comment:
@vchoi-hdfgroup or @jhendersonHDF , so does nvcc require only for this part? Can the rest of software stack (Argotbot / Mercury) be compiled with gcc/openmpi?

Reply:
nvcc is only for compiling this gpu-margo-p2p-b2.cu.  Rest is compiled with gcc/openmpi.